aws-datastore/src/test/resources/update-meeting.json

```diff
@@ -0,0 +1,14 @@
+{"query":"mutation UpdateMeeting($input:UpdateMeetingInput!) {
```

Comment:
Hm, I don't think this is valid JSON.

Maybe either - 

1. Rename the file to `update-meeting.txt`, or;
2. Rework the string value here so that the embedded GraphQL is properly escaped? (at least `{`, possibly other chars?)

Reply:
Renamed to `update-meeting.txt`